src/transactions/feed/EarnFeedItem.tsx

```diff
@@ -0,0 +1,160 @@
+      style={textStyle}
+      testID={`EarnFeedItem/${transaction.__typename}-amount-${isLocal ? 'local' : 'crypto'}`}
+    />
+  )
+}
+
+interface AmountProps {
+  transaction: EarnWithdraw | EarnDeposit | EarnClaimReward
+}
+
+function Amount({ transaction }: AmountProps) {
+  return (
+    <View style={styles.amountContainer}>
+      <AmountDisplay transaction={transaction} isLocal={false} />
+      <AmountDisplay transaction={transaction} isLocal={true} />
+    </View>
+  )
+}
+
+interface Props {
+  transaction: EarnWithdraw | EarnDeposit | EarnClaimReward
+}
+
+export default function EarnFeedItem({ transaction }: Props) {
+  const { t } = useTranslation()
+  const { providerName } = getDynamicConfigParams(
+    DynamicConfigs[StatsigDynamicConfigs.EARN_STABLECOIN_CONFIG]
+  )
+  const isDeposit = transaction.__typename === 'EarnDeposit'
+  const isWithdraw = transaction.__typename === 'EarnWithdraw'
+  const isClaim = transaction.__typename === 'EarnClaimReward'
+
+  return (
+    <Touchable
+      testID={`EarnFeedItem/${transaction.transactionHash}`}
+      onPress={() => {
+        ValoraAnalytics.track(EarnEvents.earn_feed_item_select, { origin: transaction.__typename })
+        navigate(Screens.TransactionDetailsScreen, {
+          transaction: { ...transaction, status: TransactionStatus.Complete },
+        })
+      }}
+    >
+      <View style={styles.container}>
+        <TransactionFeedItemImage
+          status={transaction.status}
+          transactionType={transaction.__typename}
+          networkId={transaction.networkId}
+        />
+        <View style={styles.contentContainer}>
+          <Text style={styles.title} testID={'EarnFeedItem/title'} numberOfLines={1}>
+            {isDeposit && t('earnFlow.transactionFeed.earnDepositTitle')}
+            {isWithdraw && t('earnFlow.transactionFeed.earnWithdrawTitle')}
+            {isClaim && t('earnFlow.transactionFeed.earnClaimTitle')}
+          </Text>
+          <Text style={styles.subtitle} testID={'EarnFeedItem/subtitle'} numberOfLines={1}>
+            {isDeposit && t('earnFlow.transactionFeed.earnDepositSubtitle', { providerName })}
+            {isWithdraw && t('earnFlow.transactionFeed.earnWithdrawSubtitle', { providerName })}
+            {isClaim && t('earnFlow.transactionFeed.earnClaimSubtitle', { providerName })}
+          </Text>
+        </View>
+        <Amount transaction={transaction} />
+      </View>
+    </Touchable>
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    paddingVertical: Spacing.Small12,
+    paddingHorizontal: variables.contentPadding,
+    alignItems: 'center',
+  },
+  contentContainer: {
+    flex: 1,
+    paddingHorizontal: variables.contentPadding,
+  },
+  title: {
+    ...typeScale.labelMedium,
+  },
+  subtitle: {
+    ...typeScale.bodySmall,
+    color: Colors.gray4,
+  },
+  amountContainer: {
+    maxWidth: '50%',
+  },
+  amountTitle: {
+    ...typeScale.labelMedium,
```

Comment:
add a color

Reply:
Fixed in 16aa133988a508a628bdd9f9bde8e841595bb178!